### ISSUE TITLE
Newtab Design bonus for implementation selected

### ISSUE THREAD

**Issue by rag-hav** [opencodeiiita/Contest-Countdown#100]
This issue is to give bonus points to those whose designs have been selected for implementation!

@Laalasa @kaneki-ken260 @underoot-iota 

Add your name to chrome/design/README.md in selected for implementation, and make a pr!

**Comment by kaneki-ken260** [comment]
But in the issue it said 20 points 😪😥

**Comment by rag-hav** [comment]
@kaneki-ken260 yes.I have decided to break it in two, an unknown number will be selected for implementation and get 10 bonus and the best among them will get 10 more
